packages/gatsby-source-packagist/package.json

```diff
@@ -38,7 +38,7 @@
     "gatsby-source-filesystem": "^4.24.0"
   },
   "peerDependencies": {
-    "gatsby": "^2.4.0 || ^3.0.0",
-    "gatsby-source-filesystem": "^2.2.4"
+    "gatsby": "^4.0.0 || ^5.0.0",
+    "gatsby-source-filesystem": "^4.24.0"
```

Comment:
This here would probably be a major bump since the peerDep on gatsby-source-filesystem means that only Gatsby 5 can be used (unless we do `>=4.24.0` and not `^5.0.0`)

Reply:
Not sure I understand. Is this peer dep, incorrect same how?

Reply:
I'm removing the v2 and v3 of Gatsby as they are pretty old at this point. So this would be a new major.

If we do a major we can decide if we want to do >=4.24.0 for source plugin or only ^5.0.0

Reply:
Lets keep Gatsby 4 support.